notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,845 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # !pip install -q -r ../requirements.txt
</pre><p>I think we can remove this comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>